data/data-pipeline/data_pipeline/ipython/aggregate_burden_exploration.ipynb

```diff
@@ -0,0 +1,1432 @@
+       "</div>"
+      ],
+      "text/plain": [
+       "Definition N (communities)  False   True\n",
+       "pct_cal_score_0.9                       \n",
+       "False                       48499  18438\n",
+       "True                          175   7048"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "text/html": [
+       "<div>\n",
+       "<style scoped>\n",
+       "    .dataframe tbody tr th:only-of-type {\n",
+       "        vertical-align: middle;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th>Definition N (communities)</th>\n",
+       "      <th>False</th>\n",
+       "      <th>True</th>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>pct_cal_score_0.9</th>\n",
+       "      <th></th>\n",
+       "      <th></th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>False</th>\n",
+       "      <td>0.653978</td>\n",
+       "      <td>0.248625</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>True</th>\n",
+       "      <td>0.002360</td>\n",
+       "      <td>0.095038</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "Definition N (communities)     False      True\n",
+       "pct_cal_score_0.9                             \n",
+       "False                       0.653978  0.248625\n",
+       "True                        0.002360  0.095038"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "for cutoff in [0.65, 0.8, 0.825, 0.85, 0.9]:\n",
+    "\n",
+    "    usa[\"pct_cal_score_\" + str(cutoff)] = usa[\"pct_cal_score\"] >= cutoff\n",
+    "    display(\n",
+    "        pd.crosstab(\n",
+    "            usa[\"pct_cal_score_\" + str(cutoff)],\n",
+    "            usa[\"Definition N (communities)\"],\n",
+    "        )\n",
+    "    )\n",
+    "    display(\n",
+    "        pd.crosstab(\n",
+    "            usa[\"pct_cal_score_\" + str(cutoff)],\n",
+    "            usa[\"Definition N (communities)\"],\n",
+    "            normalize=True,\n",
+    "        )\n",
+    "    )"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fa47f4f8-16bd-47ec-b2a5-e20fe2d83c83",
+   "metadata": {},
+   "source": [
+    "## Does it square with calenvironscreen? "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "5498bbd7-b40f-4e17-9dd4-c102f071b250",
+   "metadata": {},
+   "outputs": [],
+   "source": [
```

Comment:
This is a super nitty comment but maybe combine all these cells that don't have any outputs? They don't really need to be in separate cells, right?

Reply:
I disagree with this and think separating cells based on logical value is important.